src/kaleidoscope/device/technomancy/Atreus.h

```diff
@@ -103,10 +94,11 @@ class Atreus: public kaleidoscope::hardware::ATMegaKeyboard {
     R3C0, R3C1, R3C2, R3C3, R3C4, R3C6,   R3C7, R3C8, R3C9, R3C10, R3C11
 }
 }
-}
 
-#include "kaleidoscope/hardware/key_indexes.h"
+typedef kaleidoscope::device::technomancy::Atreus Device;
```

Comment:
Does it matter that this go at the end? It feels like it should be first.

Reply:
It has to be in the kaleidoscope namespace, after `kaleidoscope::device::technomancy` has been declared. So it can't be at the top of the file, sadly. Nor can it be outside of the `namespace kaleidoscope {}` block, because the compiler would then tell me that `kaleidoscope::Device` can only be declared within that namespace.